### PR TITLE
Make the user parameter optional.

### PR DESCRIPTION
**Pull request by fpeterschmitt**
If given, stay with the default behavior (su - <user> -c cmd), otherwise use the
currently loged-in user.

Also update the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,20 +9,24 @@ See also
 Installation
 ------------
 
-    curl https://raw.github.com/jistr/ansible-dconf/master/dconf.py > ~/ansible_dir/library/dconf
+    curl https://raw.githubusercontent.com/Leryan/ansible-dconf/master/dconf.py > ~/ansible_dir/library/dconf
```
Comment (jistr) [inline README.md]: Please keep the github username unchanged (jistr) in the pull request.

File: README.md [hunk 2]
```diff
@@ -9,20 +9,24 @@ See also
...
     - name: gnome-terminal default-show-menubar
-      dconf: user=jistr
-             key=/org/gnome/terminal/legacy/default-show-menubar
-             value=false
+      dconf:
+        user: 'jistr'
+        key: '/org/gnome/terminal/legacy/default-show-menubar'
+        value: 'false'
+
+The `user` parameter is optional, using the currently connected user. If set, it uses `su - <user>` to switch.
```
Comment (jistr) [inline README.md]: "using the currently connected user" => "using the currently connected user if omitted"

This might sound a bit clearer?

**Review by jistr — CHANGES_REQUESTED** [review]
Thanks, this looks good! I have 2 small requests for change, and then i think it's good to merge.

File: README.md (hunk 1 above)
Comment (fpeterschmitt) [reply README.md]: Woops, wrong copy paste :)

File: README.md (hunk 2 above)
Comment (fpeterschmitt) [reply README.md]: Yes, english isn't my primary language and i still struggle to keep sentences clear, no problem with that!

**Review by jistr — APPROVED** [review]

**Comment by jistr** [comment]
Thanks!

**Pull request merged**